cluster/engine.go

```diff
@@ -491,6 +492,14 @@ func (e *Engine) updateSpecs() error {
 		e.lastError = message
 		return fmt.Errorf(message)
 	}
+
+	if info.SystemTime != "" {
+		engineTime, _ := time.Parse(time.RFC3339Nano, info.SystemTime)
+		e.DeltaDuration = time.Now().UTC().Sub(engineTime)
```

Comment:
I will add the threshold in updateSpecs. So Swarm will not enter an if when receiving a request